tests/unit/brevo.t

```diff
@@ -0,0 +1,46 @@
+use ProductOpener::PerlStandards;
+use Test::More;
+use Test::MockModule;
+use HTTP::Response;
+use HTTP::Headers;
+use JSON;
+use Test::Fake::HTTPD qw/run_http_server/;
+use ProductOpener::Brevo qw/:all/;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use File::Temp ();
+
+# Mock the LWP::UserAgent module
+my $mocked_ua = Test::MockModule->new('LWP::UserAgent');
+$mocked_ua->mock(
+    'request' => sub {
+        my ($request) = @_;
```

Comment:
you could instead define $request above, so that after the test you can verify you get what you expected:

```suggestion
my $request;
mock(...
	($request) = @_;
	...
);
...
# test
...
is_deeply($request, {....});
```